cypress/integration/supervision/letters/create-letter.spec.js

```diff
@@ -0,0 +1,88 @@
+const createClient = () => {
+  return cy.fixture('client/minimal.json').then(client => {
+    cy.postToApi('/api/v1/clients', client)
+      .its('body')
+      .then(res => {
+        cy.wrap(res.id).as('clientId');
+      });
+  });
+};
+
+const createOrder = () => {
+  return cy.get('@clientId').then(clientId => {
+    cy.fixture('order/minimal.json').then(order => {
+      cy.postToApi('/api/v1/clients/' + clientId + '/orders', order)
+        .its('body')
+        .then(res => {
+          cy.wrap(res.caseRecNumber).as('courtReference');
+          cy.wrap(res.id).as('orderId');
+        });
+    });
+  });
+};
+
+const getIframeDocument = () => {
+  return cy.get('iframe[id="editor_ifr"]', {timeout:60000}).its('0.contentDocument').should('exist')
```

Comment:
Question the timeout will that run for each time in the CI and will it last for 60000 milliseconds?

Reply:
It will run for a max of 60000 milliseconds but continuously retries until the first time it is resolved so it much faster.